### DISCUSSION TITLE
trans/hello-pulumi-from-jeo-duffy/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#361] [Announcements]
# trans/hello-pulumi-from-jeo-duffy/

2018 年 6 月 18 日 Joe Duffy 在他的博客中宣布开源了云原生编程语言 Pulumi。

https://jimmysong.io/trans/hello-pulumi-from-jeo-duffy/

**Comment by TechQuery** [comment]
明明是一个 NPM 包，充其量是兼容性框架，偏说是新语言……

**Reply by rootsongjc** [reply]
市场营销策略，不过 Pulumi 这些年好像发展还不错。

**Reply by TechQuery** [reply]
> 市场营销策略

这种营销策略成功劝退了我这样的用户。

> 不过 Pulumi 这些年好像发展还不错。

在国内从没听说过……
